csharp/Svix/SvixOptions.cs

```diff
@@ -2,17 +2,24 @@ namespace Svix
 {
     public class SvixOptions
     {
+        [Obsolete("BaseUrl is deprecated, please use ServerUrl instead.")]
         public string BaseUrl { get; }
+        public string ServerUrl { get; }
         public List<int> RetryScheduleMilliseconds { get; } = [50, 100, 200];
         public int TimeoutMilliseconds { get; } = 15000;
 
+        /// <param name="serverUrl">The server URL to connect to.</param>
+        /// <param name="baseUrl">[Deprecated] Please use serverUrl parameter instead.</param>
+#pragma warning disable CS0618
         public SvixOptions(
-            string baseUrl,
+            string serverUrl,
```

Comment:
This should be optional as well because:
1. Otherwise it'll break for people that use baseUrl as named (this won't be set).
2. It's optional anyway, we have a default...